### ISSUE TITLE
OCR content recognition fails in some NVDA dialogs

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#11243]
### Steps to reproduce:
1. Open NVDA
2. Open either the interpunctuations and symbols dialog or the add-ons manager dialog
3. Press windows+up arow to try to maximize the windows
4. Press nvda+r to perform OCR recognition
5. Press windows+down arrow once
6. Press nvda+r again

### Actual behavior:
NVDA cannot perform the content recognition in step 4. Following error is in the log:
```
IO - inputCore.InputManager.executeGesture (21:02:22.033) - winInputHook (1988):
Input: kb(laptop):NVDA+r
IO - inputCore.logTimeSinceInput (21:02:22.053) - MainThread (4612):
0.020 sec since input
IO - speech.speak (21:02:22.053) - MainThread (4612):
Speaking ['Erkennung läuft']
IO - speech.manager.SpeechManager._pushNextSpeech (21:02:22.053) - MainThread (4612):
Sending to synth: ['Erkennung läuft  ', IndexCommand(2994)]
ERROR - scriptHandler.executeScript (21:02:22.093) - MainThread (4612):
error executing script: <bound method GlobalCommands.script_recognizeWithUwpOcr of <globalCommands.GlobalCommands object at 0x03FAB790>> with gesture 'NVDA+r'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 205, in executeScript
  File "globalCommands.pyc", line 2443, in script_recognizeWithUwpOcr
  File "contentRecog\recogUi.pyc", line 150, in recognizeNavigatorObject
  File "contentRecog\uwpOcr.pyc", line 108, in recognize
OSError: [WinError -529697949] Windows Error 0xe06d7363
```
In step 6 the content recognition works again.

### Expected behavior:
NVDA should perform the OCR recognition without error also in step 4.
I think this fails due to bad implementation of window coordinates or offsets. @JulienCochuyt do you have any thoughts here?

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Version: alpha-20272,893535fc

#### Windows version:
Windows 10 1909 update

#### Name and version of other software in use when reproducing the issue:
n/a

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
earlier versions show this behavior also. But I don't know which is the earliest one.

#### If addons are disabled, is your problem still occuring?
yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
yes, no effect.

**Comment by Adriani90** [comment]
I think the column size in these dialogs are not implemented as dynamic values, maybe this is causing this problem?

**Comment by JulienCochuyt** [comment]
@Adriani90, I'm glad you noticed the Symbols Pronunciation dialog can now be maximized (introduced in 2019.3)
I'm sorry but I could not reproduce the faulty behavior you're experiencing.
Tried under Windows 10 1909 with installed 2020.1, portable alpha-20272,893535fc, and latest master (893535f) running from source.

In step 2, did you open those dialogs from the NVDA menu or used a custom gesture?
I take for granted you did not press any other key between steps 2, 3 and 4, is that right?

**Comment by feerrenrut** [comment]
This error seems to be `STATUS_CPP_EH_EXCEPTION`: "C++ exception handling exception"

Bit of a stab in the dark, but I would guess that a likely culprit might be the OCR language support. Perhaps you have had an update recently that has caused a problem with it? Check the language selected in NVDA settings for OCR?

Either way, it would be good if NVDA provided a clearer and more user friendly error if possible.

**Comment by Adriani90** [comment]
This is not reproducible anymore in NVDA 2022.4. Closing.
